samples/apps/logging/logging.cpp

```diff
@@ -1072,6 +1072,185 @@ namespace loggingapp
           ccf::endpoints::ExecuteOutsideConsensus::Locally)
         .install();
 
+      static constexpr auto get_historical_multi_path =
+        "/log/private/historical/multi";
```

Comment:
I think `sparse` would be a little more accurate than `multi`, which could potentially include ranges, but naming etc.